operators/src/engine/query.rs

```diff
@@ -1,50 +1,12 @@
 use std::sync::Arc;
 
-/// A spatio-temporal rectangle for querying data
-use geoengine_datatypes::primitives::{
-    AxisAlignedRectangle, BoundingBox2D, SpatialPartition2D, SpatialPartitioned, SpatialResolution,
-    TimeInterval,
+pub use geoengine_datatypes::primitives::{
+    PlotQueryRectangle, QueryRectangle, RasterQueryRectangle, VectorQueryRectangle,
```

Comment:
You should actually change the imports instead of re-exporting it.

The re-exporting could be done on a separate PR to get this merged.
But we should do it.

Reply:
fc8d44e